src/main/java/buildcraft/compat/forestry/pipes/gui/GuiPropolisPipe.java

```diff
@@ -237,6 +250,11 @@ public ToolTip getToolTip() {
 
         @Override
         public boolean handleMouseClick(int mouseX, int mouseY, int mouseButton) {
+            ItemStack neiDraggedStack = ItemPanels.itemPanel.draggedStack;
+            if (neiDraggedStack != null) {
+                // drag is handled by #handleDragNDrop
+                return false;
+            }
```

Comment:
This check should be performed only if NEI is loaded (Forestry module of the mod doesn't requires NEI).

Reply:
fixed and tested without NEI plugin loaded